lib/analyzer/static-analyzer.ts

```diff
@@ -61,6 +72,11 @@ export async function analyze(
     getRpmDbFileContent(archiveLayers, options.tmpDirPath),
   ]);
 
+  let distrolessAptFiles: string[] = [];
+  if (options.distroless) {
+    distrolessAptFiles = getAptFiles(archiveLayers);
```

Comment:
does this has to be wrapped in an if statement?

Reply:
if we want this change to be rolled in a very controlled environment, then yes in my opinion